### PR TITLE
Fixes failures on buildpack names

### PR DESCRIPTION
**Pull request by TisVictress**
Updates tests to account for [buildpack name changes](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0050-buildpack-rename.md).

### CODE REVIEW

**Comment by TisVictress** [comment]
@paketo-buildpacks/go-maintainers Can someone confirm it's okay to remove the go dep tests? (especially since go dep tests were [removed](https://github.com/paketo-buildpacks/go/commit/ba9bd3c83dd0831afea75804346464eec32eaeaa) from the buildpack)

File: go/go_test.go [hunk 1]
```diff
@@ -88,12 +88,12 @@ func testGoWithBuilder(builder string) func(*testing.T, spec.G, spec.S) {
...
-					Expect(logs).To(ContainLines(ContainSubstring("Paketo Go Build Buildpack")))
+					Expect(logs).To(ContainLines(ContainSubstring("Paketo Buildpack for Go Distribution")))
+					Expect(logs).To(ContainLines(ContainSubstring("Paketo Buildpack for Go Build")))
 
-					Expect(logs).NotTo(ContainLines(ContainSubstring("Paketo Go Mod Vendor Buildpack")))
-					Expect(logs).NotTo(ContainLines(ContainSubstring("Paketo Dep Buildpack")))
-					Expect(logs).NotTo(ContainLines(ContainSubstring("Paketo Dep Ensure Buildpack")))
+					Expect(logs).NotTo(ContainLines(ContainSubstring("Paketo Buildpack for Go Mod Vendor")))
+					Expect(logs).NotTo(ContainLines(ContainSubstring("Paketo Buildpack for Dep")))
+					Expect(logs).NotTo(ContainLines(ContainSubstring("Paketo Buildpack for Dep Ensure")))
```
Comment (fg-j) [inline go/go_test.go:96]:
```suggestion
```

Comment (TisVictress) [reply go/go_test.go]: 😆 I knew I was gonna miss something! Thanks @fg-j

**Review by ForestEckhardt — DISMISSED** [review]

**Review by fg-j — DISMISSED** [review]

**Review by robdimsdale — DISMISSED** [review]

**Comment by robdimsdale** [comment]
@TisVictress I don't see any python test changes, but I see those tests failing for the same reason. Is there a reason we didn't update these tests? Am I missing something?

**Review by robdimsdale — APPROVED** [review]

**Comment by sophiewigmore** [comment]
I noticed that Dan also opened https://github.com/paketo-buildpacks/samples/pull/396 a few days ago, should that PR be closed out?

**Comment by TisVictress** [comment]
Closing since we're going to submit a PR for each suite to avoid failed checks.

**Pull request closed without merging**